pimd/pim_pim.c

```diff
@@ -253,6 +253,19 @@ int pim_pim_packet(struct interface *ifp, uint8_t *buf, size_t len)
 					      pim_msg_len - PIM_MSG_HEADER_LEN);
 		break;
 	case PIM_MSG_TYPE_JOIN_PRUNE:
+		if (ip_hdr->ip_dst.s_addr != qpim_all_pim_routers_addr.s_addr) {
+			char dst_str[INET_ADDRSTRLEN];
+			char src_str[INET_ADDRSTRLEN];
+
+			pim_inet4_dump("<dst?>", ip_hdr->ip_dst, dst_str,
+				       sizeof(dst_str));
+			pim_inet4_dump("<src?>", ip_hdr->ip_src, src_str,
+				       sizeof(src_str));
+			zlog_warn(
+				"%s: Ignoring Pkt. Unexpected IP destination %s for join (Expected: all_pim_routers_addr) from %s",
+				__PRETTY_FUNCTION__, dst_str, src_str);
```

Comment:
standard is now __func__ instead of __PRETTY_FUNCTION__

Reply:
Shall change it